src/io.jl

```diff
@@ -1,16 +1,37 @@
 #
 # Bayes Net
 #
-Base.showable(::MIME"image/svg+xml", bn::BayesNet) = success(`lualatex -v`)
+lualatex_available() = success(`lualatex -v`)
```

Comment:
I'd recommend putting a try/catch around the call to success. It would return false if it fails. Then you don't need try/catch in showable and plot.